### PR TITLE
[Method References 2] LPS-167730 Remove method reference in class PortletRegistryImpl

### PR DESCRIPTION
**Pull request by dantewang**
@W2jx I only modified your commit and messages, no code change.

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@dantewang No regressions.

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/processor/PortletRegistryImpl.java [hunk 1]
```diff
@@ -127,12 +118,22 @@ public List<String> getFragmentEntryLinkPortletIds(
...
+	}
+
+	@Override
+	public void registerAlias(String alias, String portletName) {
+		_aliasPortletNames.put(alias, portletName);
+	}
+
+	@Override
+	public void unregisterAlias(String alias, String portletName) {
+		_aliasPortletNames.remove(alias, portletName);
```
Comment (tinatian) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/processor/PortletRegistryImpl.java:136]: @W2jx Please do not do `remove(alias, portletName)`, do `remove(alias)` directly, I did not see any reason you have to do this.

**Comment by tinatian** [comment]
@dantewang @W2jx Actually all the impl of FragmentCollectionContributor should have its dependency to its portlet, if you do not have the portlet ready, how can you have a fragment version of it? 

We do not have the dependency for all is because we only see those related to `SiteNavigationMenuPortlet` are failed.

Also, we should have not a separate model for the contributors as it depend on corresponding portlet, it should be defined in the same bundle as its portlet.

Tina

**Pull request closed without merging**